tutorials/assets/CT_to_USD/README.md

```diff
@@ -0,0 +1,189 @@
+
+```bash
+python -m monai.bundle run --config_file configs/inference_all.json
+```
+
+### 4️⃣ **Visualize generated CT Data**
+
+Install Slicer SDK or another application to view the CT data and labelmap.
+
+![MAISI_CT_data](images/Slicer_view_SDG.png)
+
+---
+
+## 🤔 Why Convert a CT Dataset from NIfTI or DICOM to USD?
+
+Medical imaging data, such as CT scans, are typically stored in formats like **NIfTI (.nii, .nii.gz)** or **DICOM (.dcm)**. These formats are well-suited for clinical and research workflows, as they efficiently store volumetric (3D) data and associated metadata. However, they are not directly compatible with 3D simulation, visualization, or robotics platforms like NVIDIA IsaacSim.
+
+**Universal Scene Description (USD)** is a powerful, extensible 3D file format developed by Pixar and widely adopted in the visual effects, animation, and simulation industries. USD is designed for efficient scene representation, asset interchange, and real-time rendering.
+
+### 🎯 Key Reasons for Conversion:
+
+- **🔄 Interoperability with Simulation Platforms:**
+  IsaacSim and other robotics/graphics tools natively support USD for importing, manipulating, and rendering 3D assets. Converting medical data to USD enables seamless integration into these environments.
+
+- **🔲 Mesh Representation:**
+  NIfTI and DICOM store volumetric data (voxels), but simulation and visualization platforms require surface meshes (e.g., OBJ, STL, or USD) to represent anatomical structures. The conversion process extracts and processes these meshes from the volumetric data.
+
+- **⚡ Efficient Rendering and Manipulation:**
+  USD supports hierarchical scene graphs, material definitions, and efficient rendering pipelines, making it ideal for interactive applications, simulation, and digital twin workflows.
+
+- **📊 Rich Metadata and Structure:**
+  USD allows for the inclusion of semantic labels, hierarchical organization, and physical properties, which are essential for robotics, AI training, and advanced visualization.
+  > 📚 **Learn more about USD:** [NVIDIA OpenUSD Learning Path](https://www.nvidia.com/en-us/learn/learning-path/openusd/)
+
+### 📋 Summary
+
+Converting CT datasets from NIfTI or DICOM to USD is necessary to:
+- ✅ Enable use in simulation and robotics platforms like IsaacSim
+- ✅ Transform volumetric medical data into usable 3D surface meshes
+- ✅ Leverage the advanced features and performance of the USD ecosystem
+
+---
+
+## 🔄 Conversion to USD Format
+The tool implements the complete conversion workflow:
+1. **NRRD → NIfTI:** Format conversion for medical data
+2. **NIfTI → Mesh:** Surface extraction with smoothing and reduction
+3. **Mesh → USD:** Final format suitable for IsaacSim and 3D visualization
+
+## 🛠️ Setup NRRD to USD Converter Tool
+
+To set up the environment and install all required dependencies for the CT-to-USD conversion workflow, run the following commands in your terminal.
+
+```bash
+cd tutorials/assets/CT_to_USD && uv sync
```

Comment:
The project doesn't use `uv` to manage dependency. Is this possible to install with robotic_ultrasound conda env?